DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/RegistrationJourneyAccessibilityTests.cs

```diff
@@ -130,6 +130,31 @@ public void Registration_by_centre_journey_with_set_password_has_no_accessibilit
             summaryResult.Violations.Should().BeEmpty();
         }
 
+        [Fact]
+        public void Register_at_new_centre_journey_has_no_accessibility_errors()
+        {
+            // given
+            var registerUrl = "/RegisterAtNewCentre";
+            Driver.LogUserInAsAdminAndDelegate(BaseUrl);
+
+            // when
+            Driver.Navigate().GoToUrl(BaseUrl + registerUrl);
+            var registerResult = new AxeBuilder(Driver).Analyze();
+            Driver.SelectDropdownItemValue("Centre", "2");
+            Driver.SubmitForm();
+
+            var learnerInformationResult = new AxeBuilder(Driver).Analyze();
+            Driver.SubmitForm();
+            Driver.LogOutUser(BaseUrl);

```

Comment:
If you don't log out, the user will stay logged in for other tests in the file, and fail because the other register journeys kick you out if you are already logged in